buildSrc/bump-version.js

```diff
@@ -57,8 +57,10 @@ async function run({ platform }) {
  * @param newVersionString {string}
  */
 async function bumpIosVersion(newVersionString) {
+	const calendarInfoPlistName = "app-ios/calendar/Info.plist"
```

Comment:
Should the calendar and mail apps always follow the same version scheme? Or would it make more sense to split it into two functions?

Reply:
Yes, they should. Otherwise we'll have problems with deactivating old clients and model versions